frontend/public/components/overview/_overview.scss

```diff
@@ -60,7 +60,8 @@ $overview-sidebar-width: 550px;
   padding-top: 5px;
 
   .co-m-pane__heading {
-    margin: 0 20px 25px;
+    font-size: var(--pf-global--FontSize--xl);
+    margin: 0 20px 16px;
```

Comment:
Shouldn't we be moving to PF spacer variables?

Reply:
I suspect that is doable. I was making an explicit change, but I can adjust to align better with of spacers.

Reply:
Hmm, not sure there is a `20px` spacer...

- `var(--pf-global--spacer--md)` is `16px` (`1rem`)
- `var(--pf-global--spacer--lg)` is `24px` (`1.5rem`)

Reply:
@serenamarie125 `24px` to the left and right of the header. Is this okay? I think not due to the alignment with the sub-headers, but just wanted to verify.

![Screen Shot 2019-11-18 at 9 03 12 AM](https://user-images.githubusercontent.com/8126518/69058781-51ed8180-09e2-11ea-8d98-ac0345473d5d.png)
![Screen Shot 2019-11-18 at 9 03 27 AM](https://user-images.githubusercontent.com/8126518/69058783-51ed8180-09e2-11ea-80e7-96ea070f69ce.png)

Reply:
@andrewballantyne use 20px because we need to keep alignment with the rest of the sidebar for now.

Reply:
Fixed